charts/dt-kubewatch/values.yaml

```diff
@@ -6,4 +6,7 @@ kubewatch:
       cpu: 100m
     limits:
       memory: 128Mi
-      cpu: 100m
\ No newline at end of file
+      cpu: 100m
+  image:
+    name: quay.io/devtron/kubewatch
```

Comment:
Replace image with repository and make corresponding changes in the template

Reply:
It's done